addons/stock/report/report_return_label.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template id="stock.report_return_label">
+        <t t-call="web.html_container">
+            <t t-call="web.external_layout">
+                <div class="page">
+                    <div class="oe_structure"/>
+                    <div class="row mt8">
+                        <div class="col-8">
+                            <h2>
+                                <span>RETURN OF <t t-out="docs[0].name"/></span>
```

Comment:
Don't use doc[0] do an iteration over docs to print the return label for each picking selected. In case of multi select the report is not working currently